### PR TITLE
Iss37

### LINKED ISSUE #37: Clean up the initial ESN notebooks
This issue can be closed when the following notebooks have been "cleaned" up by replacing some blocks with the wrapper functions created in PR #35.

- [x] ``esn_demand``
- [x] ``esn_solarpower``
- [x] ``esn_windpower``

### LINKED ISSUE #39: Create a function to generate formatted string with model params
This issue can be closed when a function has been added that generates a formatted string from the model params.

### PR DESCRIPTION
**Pull request by samgdotson**
This PR adds three new notebooks 
* ``clean_demand``
* ``clean_solar``
* ``clean_wind``
to replace the original notebooks (which have been removed). 

I also added a ``param_string`` function to return a descriptive string of model parameters.

Closes #39 
Closes #37 

Notes: 
1. If you open ``clean_wind``, it will show an error in the training length optimization. This was an input error, and not a code error. I tried training on more data than was available. The values should be the same because it used the same random seed as the original notebook. **It took almost 3 hours for this notebook to run. I'm not going to run it again for a cosmetic issue at this time.**
2. @pep8speaks will complain about the line in ``param_string`` that is greater than 80 characters. There is no way to shorten this line. Let it complain.

### CODE REVIEW

**Comment by robfairh** [comment]
These are suggestions. Some of the labels have changed.

* in clean_demand second figure change rho by \rho symbol
* in clean_demand third figure change n_reservoir by Reservoir Size
* in clean_demand fourth figure change trainlen by Training length

Other than that, it looks good.

**Comment by samgdotson** [comment]
@robfairh right now, the hyper-parameter plots take the labels from the hyper parameters being optimized. Specified in ``optimizers.py``. I can make an issue to improve the style of the plots. Otherwise, consider merging?

**Comment by LukeSeifert** [comment]
Consider changing "4. Demonstrate the effect of training length on accuracy." to "4. A demonstration of training length's effect on accuracy." to be more consistent with the other items' wording.
I couldn't find anything else, looks good!

**Comment by samgdotson** [comment]
Great! @LukeSeifert consider smashing the merge button?
